Makefile

```diff
@@ -27,6 +27,9 @@ flow:
 fix:
 	./node_modules/.bin/eslint packages/ --format=codeframe --fix
 
+prettify:
+	./node_modules/.bin/prettier --trailing-comma all --write "packages/**/src/*.js"
```

Comment:
Yeah that is totally fair, I didn't fully think that through initially. I'm fine with trying out 80 for now, we can always increase it later.

One thing we'll have to consider is that we have a bunch of code snippets inside template literals that are currently formatted for 110 and those won't be reformatted, so those will stick out. If we reduce ESLint to 80 we'll have to go mark all of those. I guess we could just leave the max at 110 for now and rely on prettier for everything else?